### PR TITLE
Refine rocksmq

### PR DESCRIPTION
**Pull request by aoiasd**
relate: https://github.com/milvus-io/milvus/issues/24812

### CODE REVIEW

File: internal/mq/mqimpl/rocksmq/server/rocksmq_impl.go [hunk 1]
```diff
@@ -166,7 +167,7 @@ func NewRocksMQ(name string, idAllocator allocator.Interface) (*rocksmq, error)
 	// L0:No Compression
 	// L1,L2: ZSTD
 	optsKV.SetNumLevels(3)
-	optsKV.SetCompressionPerLevel([]gorocksdb.CompressionType{0, 7, 7})
+	optsKV.SetCompressionPerLevel([]gorocksdb.CompressionType{0, 1, 7})
```
Comment (xiaofan-luan) [inline internal/mq/mqimpl/rocksmq/server/rocksmq_impl.go:170]: don't play this trick.
this require too much dependency

**Comment by xiaofan-luan** [comment]
most of the part looks good to me

File: internal/mq/mqimpl/rocksmq/server/rocksmq_impl.go (hunk 1 above)
Comment (xiaofan-luan) [reply internal/mq/mqimpl/rocksmq/server/rocksmq_impl.go:170]: and we should let this to be configurable

Comment (aoiasd) [reply internal/mq/mqimpl/rocksmq/server/rocksmq_impl.go:170]: > don't play this trick. this require too much dependency

Type 1 means snappy, was default compression type, will not require new dependency. And I will make this configurable.

**Comment by aoiasd** [comment]
rerun ut

**Comment by aoiasd** [comment]
rerun ut

**Pull request merged**